cmd/mattermost/commands/webhook.go

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package commands
+
+import (
+	"github.com/mattermost/mattermost-server/model"
+	"github.com/spf13/cobra"
+)
+
+var WebhookCmd = &cobra.Command{
+	Use:   "webhook",
+	Short: "Management of webhooks",
+}
+
+var WebhookListCmd = &cobra.Command{
+	Use:     "list",
+	Short:   "List webhooks",
+	Long:    "list all webhooks",
+	Example: "  webhook list myteam",
+	RunE:    listWebhookCmdF,
+}
+
+func listWebhookCmdF(command *cobra.Command, args []string) error {
+	app, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer app.Shutdown()
+
+	var teams []*model.Team
+	if len(args) < 1 {
+		// If no team is specified, list all teams
+		teams, err = app.GetAllTeams()
+		if err != nil {
+			return err
+		}
+	} else {
+		teams = getTeamsFromTeamArgs(app, args)
+	}
+
+	for i, team := range teams {
+		if team == nil {
+			CommandPrintErrorln("Unable to find team '" + args[i] + "'")
+			continue
+		}
+
+		incomingResult := app.Srv.Store.Webhook().GetIncomingByTeam(team.Id, -1, -1)
+		outgoingResult := app.Srv.Store.Webhook().GetOutgoingByTeam(team.Id, -1, -1)
+
+		if result := <-incomingResult; result.Err == nil {
+			CommandPrettyPrintln("Incoming Webhooks:")
```

Comment:
Maybe the team name must be here, for example `fmt.Sprintf("Incomming webhooks for %s (%s):", team.DisplayName, team.Name)`. This would replace the another commend of include the team.Name in each line.

Reply:
ill do this